examples/wallet/wallet.vy

```diff
@@ -25,9 +25,9 @@ def testEcrecover(h: bytes32, v:uint256, r:uint256, s:uint256) -> address:
 # `@payable` allows functions to receive ether
 @public
 @payable
-def approve(_seq: int128, to: address, value: uint256, data: bytes[4096], sigdata: uint256[3][5]) -> bytes[4096]:
+def approve(_seq: int128, to: address, _value: uint256, data: bytes[4096], sigdata: uint256[3][5]) -> bytes[4096]:
     # Throws if the value sent to the contract is less than the sum of the value to be sent
-    assert msg.value >= value
+    assert msg.value >= _value
```

Comment:
Does this fail with the changes in this PR?

Regardless, could probably move this to a general cleanup PR